lib/fml_forms/validations.rb

```diff
@@ -10,6 +10,7 @@ module FML
   #   Given a spec like:
   #    form:
   #      title: "Validation Example"
+  #      id: 0acd2dc0b8b325eb5698e50387b79d00d77b1f10
```

Comment:
Why do ids like this? Are they auto-generated? We want FML to remain something that a non-technical person can use. Can we make the id a parameterized version of the title?